tests/_benderjs/ckeditor/static/tools.js

```diff
@@ -1080,15 +1081,15 @@
 					'setUp': 1,
 					'tearDown': 1
 				},
-				i, editorName;
+				i, editorName, editorCounter;
 
 			for ( var method in specialMethods ) {
 				if ( inputTests[ method ] ) {
 					outputTests[ method ] = inputTests[ method ];
 				}
 			}
 
-			for ( i = 0; i < editorsNames.length; i++ ) {
+			for ( i = 0, editorCounter = 0; i < editorsNames.length; i++, editorCounter = 0 ) {
```

Comment:
Looks a little unusual at first sight, but works fine and is quite straightforward and minimalistic approach which is good 👍